tutorial-week-4.md

```diff
@@ -0,0 +1,14 @@
+To be honest I have a difficult time coming up with topics for tutorial based journal entries but this one popped into my head so I figured I’d roll with it. Right now this is mostly just a stream of thoughts that I’ll flesh out over the week so bear with me if it doesn't make as much sense at first.
```

Comment:
How about a tutorial for something technical? Like how to do something in CSS, or HTML, or JS?